<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-17JJ3GD" name="Test(failed)" tests="2" failures="2" timestamp="30 Sep 2020 05:29:20 GMT" time="46.011" errors="0">
  <testcase name="login2" time="0.068" classname="com.Redbus.TestCases.LoginTestRedbus">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.Redbus.TestCases.LoginTestRedbus.login2(LoginTestRedbus.java:38)
	at TestCasefailedSuite.runner.FailedTest(runner.java:23)
... Removed 47 stack frames]]>
    </failure>
  </testcase> <!-- login2 -->
  <testcase name="loginpage2" time="0.0" classname="com.Redbus.TestCases.LoginTestRedbus">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.Redbus.TestCases.LoginTestRedbus.loginpage2(LoginTestRedbus.java:48)
	at TestCasefailedSuite.runner.FailedTest(runner.java:23)
... Removed 47 stack frames]]>
    </failure>
  </testcase> <!-- loginpage2 -->
</testsuite> <!-- Test(failed) -->
